src/connector/src/aws_utils.rs

```diff
@@ -105,6 +105,7 @@ pub fn s3_client(
         s3_config::Builder::from(&sdk_config.clone())
             .retry_config(retry_conf)
             .timeout_config(timeout_conf)
+            .force_path_style(true)
```

Comment:
Is there any potential risk here? Because IIRC Hummock only sets `force_path_style` when necessary cc. @wcy-fdu